Can't build current version, bin file is in _bin, not root repo dir

Hi, seems like this change https://github.com/cllunsford/aws-signing-proxy/blob/master/Dockerfile#L14
is breaking current builds when using `make build`:

```
docker build -t cllunsford/aws-signing-proxy:latest .
Sending build context to Docker daemon 142.9 MB
Step 1/6 : FROM scratch
 ---> 
Step 2/6 : MAINTAINER Chris Lunsford <cllunsford@gmail.com>
 ---> Using cache
 ---> 98af7f3036ff
Step 3/6 : ADD ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 ---> Using cache
 ---> 2fe3d0bf569c
Step 4/6 : ADD aws-signing-proxy /
lstat aws-signing-proxy: no such file or directory
Makefile:22: recipe for target 'dockbuild' failed
make: *** [dockbuild] Error 1
```

That fixes it:
```
--- a/Dockerfile
+++ b/Dockerfile
@@ -6,7 +6,7 @@ MAINTAINER Chris Lunsford <cllunsford@gmail.com>
 ADD ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 
 # Add executable
-ADD aws-signing-proxy /
+ADD _bin/aws-signing-proxy /
 
 # Default listening port
 EXPOSE 8080
```

--- a/Makefile
+++ b/Makefile
@@ -35,6 +35,7 @@ gobuild:
 	-e GOOS=linux  \
 	golang:${go_ver} \
 	bash -c "go get ./src/${gitrepo}/${name}/...; chown -R $$(id -u):$$(id -g) ./"
+	ln -f _bin/aws-signing-proxy
 
 clean:
 	rm -rf ./_* ca-certificates.crt aws-signing-proxy

